setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+usage="Usage: ./setup.sh </path/to/venv/>"
+
+# Check if path for venv folder provided
+if [[ -z "$1" ]]; then
+    echo -e "Missing required argument.\n$usage\n"
+    exit 1
```

Comment:
```suggestion
    return 1
```
Sourced scripts should use `return`